src/customlabels.c

```diff
@@ -117,7 +117,7 @@ static int push(custom_labels_string_t key, custom_labels_string_t value) {
 
 static int labelset_push(custom_labels_labelset_t *ls, custom_labels_string_t key, custom_labels_string_t value) {
         if (ls->count == ls->capacity) {
-                size_t new_cap = MAX(2 * cur_capacity, 1);
+                size_t new_cap = MAX(2 * ls->capacity, 1);
```

Comment:
@umanwizard please double check this, but I noticed a segfault when trying to set labels in an empty labelset without a current labelset, and this seemed pretty clearly wrong!